packages/ckeditor5-coremedia-images/src/converters.ts

```diff
@@ -79,6 +79,12 @@ export const editingDowncastXlinkHref =
   (editor: Editor, modelElementName: string): DowncastConversionHelperFunction =>
   (dispatcher: DowncastDispatcher) => {
     dispatcher.on(`attribute:xlink-href:${modelElementName}`, (eventInfo: EventInfo, data): void => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
I would prefer to type the data in line 81 like this:
````
dispatcher.on(`attribute:xlink-href:${modelElementName}`, (eventInfo: EventInfo, data: {newAttributeValue: string}): void => {
````
I'm not sure if newAttributeValue may be `null` or `undefined`